runhouse/rns/login.py

```diff
@@ -149,11 +150,11 @@ def _login_download_secrets(headers: Optional[str] = None):
     for name in secrets:
         try:
             secret = Secret.from_name(name)
-
-            download_path = secret.path or secret._DEFAULT_CREDENTIALS_PATH
-            if download_path:
-                logger.info(f"Loading down secrets for {name} into {download_path}")
-                secret.write()
+            if hasattr(secret, "path"):
+                download_path = secret.path or secret._DEFAULT_CREDENTIALS_PATH
+                if download_path:
+                    logger.info(f"Loading down secrets for {name} into {download_path}")
+                    secret.write()
```

Comment:
If the secret doesn't have a path but has a default place for env vars, do we drop the loaded values into env vars, or ignore them?

Reply:
per discussion, will save into os.environ if called from python rh.login(). if called through runhouse login app command, will save the env vars into ~/.rh/secrets/login.env and ask the user to source env vars themselves.